docs/2_4_settings.md

```diff
@@ -16,7 +16,7 @@ ReVanced Manager has settings that can be configured to your liking.
   
   Automatically fetch the latest ReVanced Patches.
 
-- ### 🪛 Allow changing patches selection
+- ### 🪛 Allow changing patch selections
 
   Allows the user to change the patches selection from the default selection.
```

Comment:
The description keeps the wording of `...patches selection` because it is easier to understand the description when it focuses on what the user can do while selecting patches for a **specific** app.